src/widgets/navigationwidget.cpp

```diff
@@ -149,6 +198,7 @@ void NavigationWidget::onParseCompleted() {
         _lastHeadingItemList[elementType] = item;
     }
     expandAll();
```

Comment:
maybe the expandAll can be dropped now since we auto move to the current node?

Reply:
Not sure about that; auto selection would only expand the current node hierarchy, not all the nodes.

Reply:
yeah, that's my point. We are auto expanding based on the cursor position, so maybe expandAll is not necessary anymore. Will also be better perf wise, since expand is a slow operation.

Either way is fine for me.